taworks/taform/models.py

```diff
@@ -78,6 +78,26 @@ class Meta:
         widgets = {'past_position_one':Textarea(attrs={'cols':80,'rows':5}), 'past_position_two':Textarea(
             attrs={'cols':80, 'rows':5}), 'past_position_three':Textarea(attrs={'cols':80, 'rows':5})}
 
+class StudentApps(ModelForm):
+    def __init__(self, *args, **kwargs):
+        super(StudentApps,self).__init__(*args, **kwargs)
+        self.fields['first_name'].widget.attrs['readonly']=True
```

Comment:
do you want to add student_id as  disabled and read only as well?

Reply:
wasn;t on front end so I excluded originally, but added for good measure